product_manufacturer/views/res_partner_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="res_partner_manufacturer_form_view" model="ir.ui.view">
+        <field name="name">res.partner.manufacturer.form.view</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='sales_purchases']//field[@name='ref']" position="before">
+                <field name="manufacturer" />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="action_res_partner_manufacturer" model="ir.actions.act_window">
+        <field name="name">Manufacturers</field>
+        <field name="res_model">res.partner</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="domain">[('manufacturer', '=', True)]</field>
+    </record>
+
+    <menuitem
+        id="menu_res_partner_manufacturer"
+        action="action_res_partner_manufacturer"
+        parent="" />
```

Comment:
I don't think a top level menu for this is justified. Top level menus are for whole applications: mrp, accounting, stock... You can add this menu under "Vendors" on the purchase app.